panel/io/ipywidget.py

```diff
@@ -81,8 +82,10 @@ def _on_widget_constructed(widget, doc=None):
         'metadata': {
             'version': __protocol_version__
         },
-        'kernel': kernel
+        'kernel': kernel,
     }
+    if Version(ipykernel.__version__) >= Version('6.22.0'):
+        args['show_warning'] = False
```

Comment:
Isn't this just hiding the problem?

Reply:
No, this is the mechanism they use internally as well.

Reply:
`ipykernel.comm.Comm` is deprecated for `comm.create_comm`, so they will remove it at some point, and things will start to fail.  If I recall correctly, they had some problems with this transition a year ago, which is why this class still exists. 

Just pasted the link to the warning message so I don't have to find it again: https://github.com/ipython/ipykernel/blob/a7d66ae2197e0d7471ba160542cf5ff7713084b5/ipykernel/comm/comm.py#L82-L83

Reply:
Calling the Comm class directly is deprecated, the class itself isn't going away.

Reply:
Actually maybe using the ipykernel BaseComm is sufficient.